client/packages/system/src/Item/api/hooks/useItemVariant/useItemVariantsConfigured.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from 'react-query';
+import { useItemGraphQL } from '../useItemApi';
+import { ITEM_VARIANTS } from '../../keys';
+
+export const useItemVariantsConfigured = () => {
+  const { api, storeId } = useItemGraphQL();
+  const { data } = useQuery({
+    queryKey: [ITEM_VARIANTS],
+    queryFn: async () => {
+      const result = await api.itemVariantsConfigured({
+        storeId,
+      });
+
+      return result.itemVariantsConfigured;
+    },
+    // Only call on page load
+    refetchOnMount: false,
```

Comment:
This means that if you set up an item variant, then immediately navigate to inbound shipment, the column won't be there... just need to refresh the page - I think this is okay? They are only configured on central server, and the no variants -> a variant exists change only happens once!

Reply:
Makes sense, but does that mean we don't really need a queryKey for this?

Reply:
Haha, probably not - that was an earlier thought, thanks